staging/src/k8s.io/client-go/tools/leaderelection/leaderelection.go

```diff
@@ -111,6 +113,12 @@ type LeaderElectionConfig struct {
 	// Callbacks are callbacks that are triggered during certain lifecycle
 	// events of the LeaderElector
 	Callbacks LeaderCallbacks
+
+	// WatchDog is the associated health checker
```

Comment:
Document that it may be nil if it's not needed?

Reply:
Done